api/v1alpha1/clusterextension_types.go

```diff
@@ -168,11 +169,11 @@ type ClusterExtensionSpec struct {
 	//
 	// When unspecified, upgrade edges across all channels will be used to identify valid automatic upgrade paths.
 	//
-	// This field follows the DNS subdomain name standard as defined in RFC
-	// 1123, with a deviation in the maximum length being no more than 48
-	// characters. This means that valid values:
-	//   - Contain no more than 48 characters
-	//   - Contain only lowercase alphanumeric characters, '-', or '.'
+	// This field follows the DNS label name standard as defined in [RFC 1123]
+	// with the addition of '.' and '_' characters being allowed.
+	// This means that valid values:
+	//   - Contain no more than 63 characters
```

Comment:
Similar question here re: removing the 15 character cushion.

Reply:
On second thought, I'm definitely good with 63 here. I can't imagine there being any cases where one would want channel names in other objects.